src/main/java/org/loose/fis/mov/services/CommService.java

```diff
@@ -66,6 +77,38 @@ public static String WordGenerator(int m){
         String result = sb.toString();
         return result;
     }
+public static void sendMail(String emailRecipient,String setSubject,String setText){
+    String host = "smtp.gmail.com";
+    Properties properties = System.getProperties();
+    properties.put("mail.smtp.host", host);
+    properties.put("mail.smtp.port", "465");
 
+    properties.put("mail.smtp.ssl.enable", "true");
+    properties.put("mail.smtp.auth", "true");
+    Session session = Session.getInstance(properties, new javax.mail.Authenticator() {
+        protected PasswordAuthentication getPasswordAuthentication() {
+            return new PasswordAuthentication("geani.gibilan", "mojojojo13");
+        }
+    });
+    session.setDebug(true);
```

Comment:
Please delete this line `session.setDebug(true);`. It floods the console with useless information.